bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2273,11 +2273,14 @@ namespace {
       Py_XDECREF(cppname);
 
       std::string pprint;
-      std::stringstream calcPrintValue;
-      calcPrintValue << "*((std::string*)" << &pprint << ") = cling::printValue((" << className << "*)"
-                     << self->GetObject() << ");";
-      gInterpreter->Calc(calcPrintValue.str().c_str());
```

Comment:
@yamaguchi1024, in order to keep the same semantics here is what we should do:
```
  auto Value = gInterpreter->CreateTemporary();
  if (gInterpreter->Evaluate("code", *Value) == success)
    pprint = gInterpreter->TInterpreterValue_SplitTypeValue(Value); // this is a new interface with a strawman proposal as name.
```

`TCling::TInterpreterValue_SplitTypeValue` should for the moment do the splitting based on strings. Cling guarantees us the following `(type) execution_result`. The type is the string enclosed between the first parens. Make sure you add a FIXME saying cling should provide proper interfaces for non-string-based type splitting.